setup.py

```diff
@@ -173,90 +206,99 @@ def visit((prefix, strip, found), dirname, names):
+    'version'            : version,
+    'description'        : "Radical Ensemble Toolkit.",
+    'long_description'   : (read('README.md') + '\n\n' + read('CHANGES.md')),
+    'author'             : 'RADICAL Group at Rutgers University',
+    'author_email'       : 'radical@rutgers.edu',
+    'maintainer'         : 'The RADICAL Group',
+    'maintainer_email'   : 'radical@rutgers.edu',
+    'url'                : 'https://github.com/radical-cybertools/radical.entk',
+    'license'            : 'MIT',
+    'keywords'           : "ensemble workflow execution",
+    'classifiers'        :  [
         'Development Status :: 4 - Beta',
         'Intended Audience :: Developers',
         'Environment :: Console',
         'License :: OSI Approved :: MIT License',
         'Programming Language :: Python',
         'Programming Language :: Python :: 2',
-        'Programming Language :: Python :: 2.5',
-        'Programming Language :: Python :: 2.6',
         'Programming Language :: Python :: 2.7',
         'Topic :: Utilities',
         'Topic :: System :: Distributed Computing',
+        'Topic :: Scientific/Engineering',
         'Operating System :: MacOS :: MacOS X',
         'Operating System :: POSIX',
         'Operating System :: Unix'
     ],
-
-    #'entry_points': {},
-
-    #'dependency_links': ['https://github.com/saga-project/saga-pilot/tarball/master#egg=sagapilot'],
-
-    'namespace_packages': ['radical', 'radical'],
-    'packages'          : find_packages('src'),
-
-    'package_dir'       : {'': 'src'},
-
-    'scripts'           : ['bin/entk-version'],
-
-
-    'package_data'      :  {'': ['*.sh', '*.json', 'VERSION', 'SDIST']},
-
-    #'install_requires'  :  ['radical.pilot', 'pika', 'pandas', 'numpy', 'matplotlib'],
-    'install_requires'  :  ['radical.utils', 'pika', 'radical.pilot',
-                            'pytest','hypothesis','sphinx'],
-
-    'zip_safe'          : False,
-
-    'data_files'        : [
-                            makeDataFiles('share/radical.entk/user_guide/scripts/', 'examples/user_guide'),
-                            makeDataFiles('share/radical.entk/simple_examples/scripts/', 'examples/simple_examples'),
-                            makeDataFiles('share/radical.entk/analytics/scripts/','examples/analytics'),
-                            makeDataFiles('share/radical.entk/advanced_examples/scripts/','examples/advanced_examples')
-                        ],
-
+    'namespace_packages' : ['radical'],
+    'packages'           : find_packages('src'),
+    'package_dir'        : {'': 'src'},
+    'scripts'            : ['bin/entk-version'],
+    'package_data'       : {'': ['*.sh', '*.json',
+                                 'VERSION', 'SDIST', sdist_name]},
+    'setup_requires'     : ['pytest-runner'],
+    'install_requires'   : ['radical.utils',
+                            'radical.pilot',
+                            'pika',
+                            'sphinx'
```

Comment:
I'm still not sure why `sphinx` is needed for installation. It is used for documentation generation.

Reply:
Ack.

Reply:
Ack. Will change.